istioctl/pkg/internaldebug/internal-debug.go

```diff
@@ -192,7 +192,11 @@ func (s *DebugWriter) PrintAll(drs map[string]*discovery.DiscoveryResponse) erro
 		}
 	}
 	if len(mappedResp) > 0 {
-		mresp, err := json.MarshalIndent(mappedResp, "", "  ")
+		rawMap := make(map[string]json.RawMessage)
+		for k, v := range mappedResp {
+			rawMap[k] = json.RawMessage(v)
```

Comment:
What if v cannot be marshalled into JSON? I think you might want to try premarshalling here. If it fails, wrap it in a string. This way, we don't marshall twice.